### ISSUE TITLE
DeprecationWarning: Ftp.raw[user](args): Use Ftp.raw('user args') instead.

### ISSUE THREAD

**Issue by SuThaw** [sergi/jsftp#226]
When I use ftp.auth method,  this deprecation warning appear. is there anyway to do authentication. my code is following.
```
const connect = (server) => new Promise((resolve,reject)=>{
	
	// create new ftp server with r	
	const ftp = new JSFtp({
		host: server.host,
		port: server.port || 21,
		user: server.user,
		pass: server.pass,
		debugMode: true
	});	
		

	ftp.on('error',function(err){
		if(err.code == 530 ) reject({status: true, err: 'invalid user credential'});
		if(err.code == 'ENOTFOUND') reject({status: true, err: 'invalid host'});
		reject({status: true,err:'criteria is not correct. cannot connect to server' });
	});

	ftp.on('connect',function(err){
		
		ftp
			.auth(server.user,server.pass,(err,data) => {
				if(err)	reject({ status: true, err: 'Error: 530 Login incorrect'});

				resolve(ftp);
			})
		// resolve(ftp);
	})
});

**Comment by joaopedroft** [comment]
It happens the same with me :(

**Comment by svenbw** [comment]
This is caused by the auth function itself for user [look at line 334](https://github.com/sergi/jsftp/blob/v1.5.5/lib/jsftp.js#L334), for password its [line 340](https://github.com/sergi/jsftp/blob/v1.5.5/lib/jsftp.js#L340) and for type it's [line 349](https://github.com/sergi/jsftp/blob/v1.5.5/lib/jsftp.js#L349).
They should be changed to `self.raw('user ' + user, function(...`, `self.raw('pass' + pass, function(...` and `self.raw('type I', function(...`.

**Comment by marcelorisse** [comment]
This was already fixed and it is merged on master: https://github.com/sergi/jsftp/commit/35a8edd6e1132898590d585a415c521df000cb58. Could anyone release a new version so we stop receiving these warnings, please?

**Comment by leizhao4** [comment]
Bump. Can we have a new release, please?

**Comment by julmot** [comment]
Receiving this warning too. Bump.

**Comment by daniel-johnson** [comment]
Receiving. Bump it up!

**Comment by adamgedney** [comment]
Same here

**Comment by simonh1000** [comment]
Although this has been closed the lines referred to by @svenbw remain unaltered in the version distributed via npm. I think the new code needs to be `npm publish`ed

**Comment by leizhao4** [comment]
Bump @sergi

**Comment by kunokdev** [comment]
+1, are moderators of this library active or should we fork ?

**Comment by juampi92** [comment]
Guys, it's just a simple push to npm publish.... please

**Comment by simonh1000** [comment]
Bump @sergi

**Comment by littledu** [comment]
Guys, it's just a simple push to npm publish.... please

**Comment by hk95** [comment]
Bump @sergi

**Comment by jazoom** [comment]
It's been a couple of months now. Is there a fork of this library that will be kept updated going forward?

**Comment by sergi** [comment]
Released v2.0.0. Really sorry for the wait, personal stuff got in the way.

**Comment by hk95** [comment]
Thank you so much for the update, @sergi.
